src/App.tsx

```diff
@@ -1,26 +1,154 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+/* eslint-disable max-len */
+
+// #region Components
+
+import { Header } from './components/Header';
+import { Footer } from './components/Footer';
+import { TodoList } from './components/TodoList';
+import { MessageError } from './components/MessageError';
+
+// #endregion
 
-const USER_ID = 0;
+// #region allImport
+
+import { Todo } from './types/Todo';
+import { getTodos, postTodos, deleteTodos } from './api/todos';
+import React, { useEffect, useRef, useState } from 'react';
+
+// #endregion
+
+export type TypeNewTodo = {
+  id: null;
+  userId: number;
+  title: string;
+  completed: boolean;
+};
+
+const USER_ID = 2881;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  // #region State
+
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [hasError, setHasError] = useState(false);
+  const [theError, setTheError] = useState<string>('');
+  const [inputValue, setInputValue] = useState<string>('');
+  const [NewTodo, setCreateNewTodo] = useState<TypeNewTodo | null>(null);
```

Comment:
```suggestion
  const [newTodo, setNewTodo] = useState<TypeNewTodo | null>(null);
```
follow to general convenion of naming state and setState function